flink-runtime/src/main/java/org/apache/flink/runtime/dispatcher/ConfigurationNotAllowedMessage.java

```diff
@@ -66,4 +66,9 @@ public static String ofConfigurationObjectRemoved(
                 "Configuration %s:%s was removed from the configuration object %s.",
                 configKey, configValue, configurationObject);
     }
+
+    public static String ofConfigurationObjectSetterUsed(
```

Comment:
nit: keep the methods simple and remove `Configuration` prefix everywhere: `ofObjectSetterUsed `

Reply:
That would make the current `ofConfigurationAdded`, `ofConfigurationRemoved` and `ofConfigurationChanged` sound strange. `ofAdded`/`ofRemoved`/`ofChanged`?

Reply:
Also true. No strong opinion here.